msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2023-10-08 11:05-0500\n"
"PO-Revision-Date: 2023-10-08 16:12\n"
"Last-Translator: \n"
"Language-Team: Dutch, Suriname\n"
"Language: nl_SR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: nl-SR\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../bin/duplicity:76
msgid "INT intercepted...exiting."
msgstr ""

#: ../bin/duplicity:84
#, python-format
msgid "GPG error detail: %s"
msgstr "GPG-foutmelding: %s"

#: ../bin/duplicity:94
#, python-format
msgid "User error detail: %s"
msgstr "Gebruikersfoutmelding: %s"

#: ../bin/duplicity:104
#, python-format
msgid "Backend error detail: %s"
msgstr "Backendfoutmelding: %s"

#: ../duplicity/tempdir.py:157
#, python-format
msgid "Using temporary directory %s"
msgstr "Tijdelijke map %s wordt gebruikt"

#: ../duplicity/tempdir.py:201
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr ""

#: ../duplicity/tempdir.py:223
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr ""

#: ../duplicity/tempdir.py:255
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Tijdelijk bestand %s wordt vergeten"

#: ../duplicity/tempdir.py:258
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "Poging om onbekend tijdelijk bestand %s te vergeten: dit is waarschijnlijk een bug."

#: ../duplicity/tempdir.py:277
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr ""

#: ../duplicity/tempdir.py:280
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr ""

#: ../duplicity/tempdir.py:285
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr ""

#: ../duplicity/backend.py:92
#, python-format
msgid "Import of %s %s"
msgstr "Import van %s %s"

#: ../duplicity/backend.py:197
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Kan backend niet initialiseren: %s"

#: ../duplicity/backend.py:384
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Backtrace van vorige foutmelding: %s"

#: ../duplicity/backend.py:401
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Opgegeven na %s pogingen. %s: %s"

#: ../duplicity/backend.py:405
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr "Poging van %s no. %s mislukt. %s: %s"

#: ../duplicity/backend.py:503
#, python-format
msgid "Reading results of '%s'"
msgstr "Resultaten van '%s' aan het lezen"

#: ../duplicity/backend.py:529
#, python-format
msgid "Writing %s"
msgstr "%s aan het schrijven"

#: ../duplicity/backend.py:570
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "Bestand %s niet lokaal gevonden na ophalen van backend"

#: ../duplicity/manifest.py:95
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr "Cruciale fout: de hostnaam van de backupbron is veranderd.\n"
"Huidige hostnaam: %s\n"
"Vorige hostnaam: %s"

#: ../duplicity/manifest.py:102
msgid "Fatal Error: Backup source directory has changed.\n"
msgstr ""

#: ../duplicity/manifest.py:111
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "De operatie wordt geannuleerd, omdat u mogelijk per ongeluk een backup probeert te maken van twee verschillende datasets naar dezelfde externe locatie, of omdat u dezelfde archiefmap probeert te gebruiken. Als dit geen vergissing is, gebruik dan de schakelaar --allow-source-mismatch om dit bericht te negeren."

#: ../duplicity/manifest.py:214
#, python-format
msgid "Found manifest volume %s"
msgstr "Manifestvolume %s gevonden"

#: ../duplicity/manifest.py:221
#, python-format
msgid "Found %s volumes in manifest"
msgstr "%s volumes gevonden in manifest"

#: ../duplicity/manifest.py:239
msgid "Manifest file '{self.fh.base if self.fh else ''}' is corrupt: "
msgstr ""

#: ../duplicity/manifest.py:256
msgid "Manifests not equal because different volume numbers"
msgstr "Manifesten niet gelijk: verschillende volumenummers"

#: ../duplicity/manifest.py:261
msgid "Manifests not equal because volume lists differ"
msgstr "Manifesten niet gelijk: volumelijsten verschillen"

#: ../duplicity/manifest.py:266
msgid "Manifests not equal because hosts or directories differ"
msgstr "Manifesten niet gelijk: hosts of mappen verschillen"

#: ../duplicity/manifest.py:426
msgid "Warning, found extra Volume identifier"
msgstr "Waarschuwing: extra volumeidentifier gevonden"

#: ../duplicity/manifest.py:452
msgid "Other is not VolumeInfo"
msgstr ""

#: ../duplicity/manifest.py:455
msgid "Volume numbers don't match"
msgstr "Volumenummers komen niet overeen"

#: ../duplicity/manifest.py:458
msgid "start_indicies don't match"
msgstr "start_indices komen niet overeen"

#: ../duplicity/manifest.py:461
msgid "end_index don't match"
msgstr "end_index komen niet overeen"

#: ../duplicity/manifest.py:466
msgid "Hashes don't match"
msgstr "Hashes komen niet overeen"

#: ../duplicity/util.py:105 ../duplicity/dup_main.py:902
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "IGNORED_ERROR: Waarschuwing: foutmelding wordt genegeerd zoals gevraagd: %s: %s"

#: ../duplicity/util.py:172
#, python-format
msgid "Releasing lockfile %s"
msgstr "Lockfile %s wordt losgelaten"

#: ../duplicity/backends/pydrivebackend.py:177
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "PyDrive-backend: meerdere bestanden heten '%s'."

#: ../duplicity/backends/multibackend.py:93
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend: kon querystring %s niet ontleden: %s "

#: ../duplicity/backends/multibackend.py:102
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "MultiBackend: querystring %s is ongeldig: meer dan één waarde voor %s"

#: ../duplicity/backends/multibackend.py:107
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr "MultiBackend: querystring %s is ongeldig: onbekende parameter %s"

#: ../duplicity/backends/multibackend.py:157
#: ../duplicity/backends/multibackend.py:162
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "MultiBackend: ongeldige waarde voor %s: %s"

#: ../duplicity/backends/multibackend.py:173
#, python-format
msgid "MultiBackend: Url %s"
msgstr "MultiBackend: URL %s"

#: ../duplicity/backends/multibackend.py:177
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MultiBackend: kon configuratiebestand %s niet laden: %s "

#: ../duplicity/backends/multibackend.py:184
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MultiBackend: gebruik store %s"

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr "MultiBackend: zet omgevingsvariabele %s = %s"

#: ../duplicity/backends/multibackend.py:202
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr "MultiBackend: registreer affiniteit voor voorvoegsel %s"

#: ../duplicity/backends/multibackend.py:243
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "MultiBackend: _put: schrijf naar store #%s (%s)"

#: ../duplicity/backends/multibackend.py:256
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "MultiBackend: poging om naar store #%s (%s) te schrijven is mislukt, probeer #%s. Foutmelding: %s"

#: ../duplicity/backends/multibackend.py:263
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr "MultiBackend: schrijven naar %s mislukt. Proces wordt afgebroken."

#: ../duplicity/backends/multibackend.py:270
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend: schrijven naar %s mislukt. Alle backend stores zijn geprobeerd en geen ervan slaagde."

#: ../duplicity/backends/multibackend.py:289
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MultiBackend: kon %s naar %s van %s niet ophalen"

#: ../duplicity/backends/multibackend.py:292
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend: ophalen van %s mislukt. Alle backend stores zijn geprobeerd en geen ervan slaagde."

#: ../duplicity/backends/multibackend.py:302
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr ""

#: ../duplicity/backends/multibackend.py:305
msgid "Exception during list of {s.backend.parsed_url.strip_auth()}: "
msgstr ""

#: ../duplicity/backends/multibackend.py:311
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:340
#: ../duplicity/backends/multibackend.py:369
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr ""

#: ../duplicity/backends/giobackend.py:108
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr ""

#: ../duplicity/backends/onedrivebackend.py:89
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:67
msgid "Missing socket or ssl python modules."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:85
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:147
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:149
#, python-format
msgid "Using WebDAV directory %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:180
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:191
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:210
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:211
#: ../duplicity/backends/webdavbackend.py:234
#, python-format
msgid "WebDAV data length: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:214
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:220
#, python-format
msgid "WebDAV redirect to: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:222
msgid "WebDAV redirected 10 times. Giving up."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:227
msgid "WebDAV missing location header in redirect response."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:232
msgid "WebDAV retry request with authentification headers."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:233
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:237
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:254
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:258
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:356
#, python-format
msgid "Creating missing directory %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:360
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:373
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:413
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:435
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:455
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/cli_data.py:100 ../duplicity/cli_data.py:161
#: ../duplicity/cli_data.py:195 ../duplicity/cli_data.py:242
#: ../duplicity/cli_data.py:345 ../duplicity/cli_data.py:363
#: ../duplicity/cli_data.py:521 ../duplicity/cli_data.py:703
#: ../duplicity/cli_data.py:932
msgid "path"
msgstr "pad"

#: ../duplicity/cli_data.py:114
msgid "Hot|Cool|Archive"
msgstr "Hot|Cool|Archive"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/cli_data.py:119 ../duplicity/cli_data.py:125
#: ../duplicity/cli_data.py:133 ../duplicity/cli_data.py:167
#: ../duplicity/cli_data.py:429 ../duplicity/cli_data.py:445
#: ../duplicity/cli_data.py:492 ../duplicity/cli_data.py:509
#: ../duplicity/cli_data.py:515 ../duplicity/cli_data.py:533
#: ../duplicity/cli_data.py:599 ../duplicity/cli_data.py:607
#: ../duplicity/cli_data.py:659 ../duplicity/cli_data.py:740
#: ../duplicity/cli_data.py:915
msgid "number"
msgstr "nummer"

#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/cli_data.py:145 ../duplicity/cli_data.py:710
#: ../duplicity/cli_data.py:949
msgid "seconds"
msgstr "seconden"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/cli_data.py:183 ../duplicity/cli_data.py:189
#: ../duplicity/cli_data.py:357 ../duplicity/cli_data.py:666
#: ../duplicity/cli_data.py:905
msgid "gpg-key-id"
msgstr "gpg-key-id"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/cli_data.py:200 ../duplicity/cli_data.py:385
#: ../duplicity/cli_data.py:955
msgid "shell_pattern"
msgstr ""

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/cli_data.py:212 ../duplicity/cli_data.py:218
#: ../duplicity/cli_data.py:278 ../duplicity/cli_data.py:391
#: ../duplicity/cli_data.py:901
msgid "filename"
msgstr "bestandsnaam"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/cli_data.py:224 ../duplicity/cli_data.py:546
#: ../duplicity/cli_data.py:987
msgid "time"
msgstr "tijd"

#: ../duplicity/cli_data.py:236 ../duplicity/cli_data.py:397
msgid "regex"
msgstr ""

#: ../duplicity/cli_data.py:339
msgid "interval"
msgstr ""

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:351 ../duplicity/cli_data.py:503
#: ../duplicity/cli_data.py:552 ../duplicity/cli_data.py:682
#: ../duplicity/cli_data.py:920 ../duplicity/argparse311.py:1773
msgid "options"
msgstr "opties"

#: ../duplicity/cli_data.py:380
msgid "imap_mailbox"
msgstr "imap_mailbox"

#: ../duplicity/cli_data.py:404
msgid "If set, an extra file with statistics in json format will be stored with each backup. It includes the statistics printed out on stdout by default. With 'collection-status' it returs these stats."
msgstr ""

#: ../duplicity/cli_data.py:411
msgid "file_descriptor"
msgstr "bestandsomschrijving"

#: ../duplicity/cli_data.py:418
msgid "log_filename"
msgstr ""

#: ../duplicity/cli_data.py:451
msgid "backup name"
msgstr "backupnaam"

#: ../duplicity/cli_data.py:558
msgid "s3_endpoint_url"
msgstr ""

#: ../duplicity/cli_data.py:626
msgid "s3_kms_key_id"
msgstr ""

#: ../duplicity/cli_data.py:632
msgid "s3_kms_grant"
msgstr ""

#: ../duplicity/cli_data.py:638
msgid "s3_region_name"
msgstr ""

#: ../duplicity/cli_data.py:644
msgid "policy"
msgstr "beleid"

#. noun
#: ../duplicity/cli_data.py:649 ../duplicity/cli_data.py:654
#: ../duplicity/cli_data.py:886
msgid "command"
msgstr "commando"

#: ../duplicity/cli_data.py:689
msgid "pem formatted bundle of certificate authorities"
msgstr "PEM-geformatteerde bundel van certificaatautoriteiten"

#: ../duplicity/cli_data.py:694
msgid "path to a folder with certificate authority files"
msgstr "pad naar een map met certificaatautoriteitbestanden"

#. abbreviation for "character" (noun)
#: ../duplicity/cli_data.py:716 ../duplicity/cli_data.py:883
msgid "char"
msgstr "char"

#: ../duplicity/cli_data.py:729
msgid "verb"
msgstr ""

#: ../duplicity/cli_data.py:747
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/cli_data.py:871
msgid "absolute_path"
msgstr "absoluut_pad"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/cli_data.py:875
msgid "alias"
msgstr "alias"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:880
msgid "bucket_name"
msgstr "bucketnaam"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/cli_data.py:891
msgid "container_name"
msgstr "containernaam"

#. noun
#: ../duplicity/cli_data.py:894
msgid "count"
msgstr "aantal"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/cli_data.py:897
msgid "directory"
msgstr "map"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/cli_data.py:910
msgid "module"
msgstr "module"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:924
msgid "other.host"
msgstr "andere.host"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:928
msgid "password"
msgstr "wachtwoord"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:936
msgid "port"
msgstr "poort"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:941
msgid "prefix"
msgstr "voorvoegsel"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/cli_data.py:945
msgid "relative_path"
msgstr "relatief_pad"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/cli_data.py:960
msgid "some_dir"
msgstr "een_map"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:966
msgid "source_path"
msgstr ""

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:971
msgid "source_url"
msgstr "bronurl"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:977
msgid "target_dir"
msgstr "doelmap"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:982
msgid "target_url"
msgstr "doelurl"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:992
msgid "user"
msgstr "gebruiker"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/cli_data.py:995
msgid "account_id"
msgstr "account_id"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/cli_data.py:999
msgid "application_key"
msgstr "applicatiesleutel"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/cli_data.py:1003 ../duplicity/dup_collections.py:546
msgid "remote"
msgstr "remote"

#: ../duplicity/cli_data.py:1006
msgid "Backends and their URL formats:"
msgstr "Backends en hun URL-formaten:"

#: ../duplicity/diffdir.py:103 ../duplicity/diffdir.py:395
#, python-format
msgid "Error %s getting delta for %s"
msgstr ""

#: ../duplicity/diffdir.py:118
#, python-format
msgid "Getting delta of %s and %s"
msgstr ""

#: ../duplicity/diffdir.py:163
#, python-format
msgid "A %s"
msgstr ""

#: ../duplicity/diffdir.py:170
#, python-format
msgid "M %s"
msgstr ""

#: ../duplicity/diffdir.py:192
#, python-format
msgid "Comparing %s and %s"
msgstr "%s en %s worden vergeleken"

#: ../duplicity/diffdir.py:200
#, python-format
msgid "D %s"
msgstr "D %s"

#: ../duplicity/librsync.py:190
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr ""

#: ../duplicity/dup_time.py:52
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr ""

#: ../duplicity/dup_time.py:59
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr ""

#: ../duplicity/cli_main.py:190
msgid "No valid action found. Will imply 'backup' because a path source was given and target is a url location."
msgstr ""

#: ../duplicity/cli_main.py:194
msgid "No valid action found. Will imply 'restore' because url source was given and target is a local path."
msgstr ""

#: ../duplicity/cli_main.py:198
#, python-brace-format
msgid "Invalid '{remainder[0]}' action and cannot be implied from the "
msgstr ""

#: ../duplicity/cli_main.py:260
#, python-brace-format
msgid "GPG binary is {config.gpg_binary}, version {gpg_version}"
msgstr ""

#: ../duplicity/cli_main.py:296
#, python-brace-format
msgid "Using archive dir: {config.archive_dir_path.uc_name}"
msgstr ""

#: ../duplicity/cli_main.py:297
#, python-brace-format
msgid "Using backup name: {config.backup_name}"
msgstr ""

#: ../duplicity/dup_main.py:108
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr ""

#: ../duplicity/dup_main.py:115
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr ""

#: ../duplicity/dup_main.py:156
msgid "PASSPHRASE variable not set, asking user."
msgstr ""

#: ../duplicity/dup_main.py:181
msgid "Retype passphrase for signing key to confirm: "
msgstr "Typ het wachtwoord voor de ondertekekeningssleutel opnieuw ter bevestiging: "

#: ../duplicity/dup_main.py:183
msgid "Retype passphrase for decryption to confirm: "
msgstr "Typ het wachtwoord ter ontsleuteling opnieuw ter bevestiging: "

#: ../duplicity/dup_main.py:186
msgid "First and second passphrases do not match!  Please try again."
msgstr "Eerste en tweede wachtwoorden komen niet overeen! Probeer het opnieuw."

#: ../duplicity/dup_main.py:193
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr ""

#: ../duplicity/dup_main.py:251
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr ""

#: ../duplicity/dup_main.py:260
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr ""

#: ../duplicity/dup_main.py:314
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d, retrying."
msgstr ""

#: ../duplicity/dup_main.py:320
#, python-format
msgid "File %s was corrupted during upload."
msgstr ""

#: ../duplicity/dup_main.py:360
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr ""

#: ../duplicity/dup_main.py:383
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr ""

#: ../duplicity/dup_main.py:386
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr ""

#: ../duplicity/dup_main.py:453
msgid "Skipped volume upload, as effectivly nothing has changed"
msgstr ""

#: ../duplicity/dup_main.py:465
#, python-format
msgid "Processed volume %d"
msgstr ""

#: ../duplicity/dup_main.py:647
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr ""

#: ../duplicity/dup_main.py:651
msgid "No signatures found, switching to full backup."
msgstr ""

#: ../duplicity/dup_main.py:665
msgid "Backup Statistics"
msgstr ""

#: ../duplicity/dup_main.py:806
#, python-format
msgid "%s not found in archive - no files restored."
msgstr ""

#: ../duplicity/dup_main.py:810
msgid "No files found in archive - nothing restored."
msgstr ""

#: ../duplicity/dup_main.py:847
#, python-format
msgid "Processed volume %d of %d"
msgstr ""

#: ../duplicity/dup_main.py:892
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr ""

#: ../duplicity/dup_main.py:895
#, python-format
msgid "Calculated hash: %s"
msgstr ""

#: ../duplicity/dup_main.py:896
#, python-format
msgid "Manifest hash: %s"
msgstr ""

#: ../duplicity/dup_main.py:947
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr ""

#: ../duplicity/dup_main.py:979
#, python-format
msgid "Verify complete: %s, %s."
msgstr ""

#: ../duplicity/dup_main.py:980
#, python-format
msgid "%d file(s) compared"
msgstr ""

#: ../duplicity/dup_main.py:981
#, python-format
msgid "%d difference(s) found"
msgstr ""

#: ../duplicity/dup_main.py:999
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr ""

#: ../duplicity/dup_main.py:1013
msgid "Found the following file(s) to delete:"
msgstr ""

#: ../duplicity/dup_main.py:1014
msgid "Run duplicity again with the --force option to actually delete."
msgstr ""

#: ../duplicity/dup_main.py:1057
msgid "There are backup set(s) at time(s):"
msgstr ""

#: ../duplicity/dup_main.py:1059
msgid "Which can't be deleted because newer sets depend on them."
msgstr ""

#: ../duplicity/dup_main.py:1063
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr ""

#: ../duplicity/dup_main.py:1076
msgid "No old backup sets found, nothing deleted."
msgstr ""

#: ../duplicity/dup_main.py:1079
msgid "Deleting backup chain(s) at time:"
msgstr ""

#: ../duplicity/dup_main.py:1089
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:1091
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:1094
#, python-format
msgid "Deleting complete signature chain %s"
msgstr ""

#: ../duplicity/dup_main.py:1096
#, python-format
msgid "Deleting complete backup chain %s"
msgstr ""

#: ../duplicity/dup_main.py:1102
msgid "Found old backup chain(s) at the following time:"
msgstr ""

#: ../duplicity/dup_main.py:1104
msgid "Rerun command with --force option to actually delete."
msgstr ""

#: ../duplicity/dup_main.py:1208
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr ""

#: ../duplicity/dup_main.py:1213
#, python-format
msgid "Unable to delete %s: %s"
msgstr ""

#: ../duplicity/dup_main.py:1248 ../duplicity/dup_temp.py:276
#, python-format
msgid "Failed to read %s: %s"
msgstr ""

#: ../duplicity/dup_main.py:1261
#, python-format
msgid "Copying %s to local cache."
msgstr ""

#: ../duplicity/dup_main.py:1309
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr ""

#: ../duplicity/dup_main.py:1314
msgid "Synchronizing remote metadata to local cache..."
msgstr ""

#: ../duplicity/dup_main.py:1327
msgid "Sync would copy the following from remote to local:"
msgstr ""

#: ../duplicity/dup_main.py:1330
msgid "Sync would remove the following spurious local files:"
msgstr ""

#: ../duplicity/dup_main.py:1374
msgid "Unable to get free space on temp."
msgstr ""

#: ../duplicity/dup_main.py:1382
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr ""

#: ../duplicity/dup_main.py:1385
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr ""

#: ../duplicity/dup_main.py:1393
msgid "Unable to get max open files."
msgstr ""

#: ../duplicity/dup_main.py:1397
#, python-format
msgid "Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr ""

#: ../duplicity/dup_main.py:1448
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr ""

#: ../duplicity/dup_main.py:1454
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr ""

#: ../duplicity/dup_main.py:1461
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr ""

#: ../duplicity/dup_main.py:1482
msgid "PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr ""

#: ../duplicity/dup_main.py:1505
#, python-format
msgid "Acquiring lockfile %s"
msgstr ""

#: ../duplicity/dup_main.py:1566
#, python-brace-format
msgid "Last {action} backup left a partial set, restarting."
msgstr ""

#: ../duplicity/dup_main.py:1570
#, python-brace-format
msgid "Cleaning up previous partial {action} backup set, restarting."
msgstr ""

#: ../duplicity/dup_main.py:1584
msgid "Last full backup date: none"
msgstr ""

#: ../duplicity/dup_main.py:1587
msgid "Last full backup is too old, forcing full backup"
msgstr ""

#: ../duplicity/dup_main.py:1640
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr ""

#: ../duplicity/asyncscheduler.py:245
#, python-format
msgid "task execution done (success: %s)"
msgstr ""

#: ../duplicity/gpginterface.py:700
#, python-brace-format
msgid "GPG process {process.pid} terminated before wait()"
msgstr ""

#: ../duplicity/robust.py:64
#, python-format
msgid "Error listing directory %s"
msgstr ""

#: ../duplicity/dup_collections.py:199 ../duplicity/dup_collections.py:210
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr ""

#: ../duplicity/dup_collections.py:235
msgid "Fatal Error: No manifests found for most recent backup"
msgstr ""

#: ../duplicity/dup_collections.py:245
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr ""

#: ../duplicity/dup_collections.py:250
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr ""

#: ../duplicity/dup_collections.py:257
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr ""

#: ../duplicity/dup_collections.py:268
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr ""

#: ../duplicity/dup_collections.py:284
msgid "Error processing remote file ({os.fsdecode(remote_file)}): {util.uexc(message)}"
msgstr ""

#: ../duplicity/dup_collections.py:286
msgid "Processing remote file {os.fsdecode(remote_file)} ({len(remote_file_buffer)})"
msgstr ""

#: ../duplicity/dup_collections.py:304
msgid "No Jsonstat file found, return enmty."
msgstr ""

#: ../duplicity/dup_collections.py:393
msgid "Preferring Backupset over previous one!"
msgstr ""

#: ../duplicity/dup_collections.py:396
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr ""

#: ../duplicity/dup_collections.py:401
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr ""

#: ../duplicity/dup_collections.py:470
msgid "Chain start time: "
msgstr ""

#: ../duplicity/dup_collections.py:471
msgid "Chain end time: "
msgstr ""

#: ../duplicity/dup_collections.py:472
#, python-format
msgid "Number of contained backup sets: %d"
msgstr ""

#: ../duplicity/dup_collections.py:474
#, python-format
msgid "Total number of contained volumes: %d"
msgstr ""

#: ../duplicity/dup_collections.py:476 ../duplicity/dup_collections.py:1277
msgid "Type of backup set:"
msgstr ""

#: ../duplicity/dup_collections.py:476 ../duplicity/dup_collections.py:1277
msgid "Time:"
msgstr ""

#: ../duplicity/dup_collections.py:476
msgid "Num volumes:"
msgstr ""

#: ../duplicity/dup_collections.py:480 ../duplicity/dup_collections.py:1283
msgid "Full"
msgstr ""

#: ../duplicity/dup_collections.py:483 ../duplicity/dup_collections.py:1285
msgid "Incremental"
msgstr ""

#: ../duplicity/dup_collections.py:544
msgid "local"
msgstr ""

#: ../duplicity/dup_collections.py:702
msgid "Collection Status"
msgstr ""

#: ../duplicity/dup_collections.py:704
#, python-format
msgid "Connecting with backend: %s"
msgstr ""

#: ../duplicity/dup_collections.py:706
#, python-format
msgid "Archive dir: %s"
msgstr ""

#: ../duplicity/dup_collections.py:708
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr ""

#: ../duplicity/dup_collections.py:711
#, python-format
msgid "Secondary chain %d of %d:"
msgstr ""

#: ../duplicity/dup_collections.py:717
msgid "Found primary backup chain with matching signature chain:"
msgstr ""

#: ../duplicity/dup_collections.py:721
msgid "No backup chains with active signatures found"
msgstr ""

#: ../duplicity/dup_collections.py:724
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr ""

#: ../duplicity/dup_collections.py:726
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr ""

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:729
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr ""

#: ../duplicity/dup_collections.py:732
msgid "No orphaned or incomplete backup sets found."
msgstr ""

#: ../duplicity/dup_collections.py:748
#, python-format
msgid "%d file(s) exists on backend"
msgstr ""

#: ../duplicity/dup_collections.py:753
#, python-format
msgid "%d file exists in cache"
msgid_plural "%d files exist in cache"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/dup_collections.py:806
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr ""

#: ../duplicity/dup_collections.py:829
msgid "Warning, found the following local orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:834
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:839
msgid "Warning, found signatures but no corresponding backup files"
msgstr ""

#: ../duplicity/dup_collections.py:843
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr ""

#: ../duplicity/dup_collections.py:847
msgid "Warning, found the following orphaned backup file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:860
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr ""

#: ../duplicity/dup_collections.py:872
#, python-format
msgid "File %s is part of known set"
msgstr ""

#: ../duplicity/dup_collections.py:875
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr ""

#: ../duplicity/dup_collections.py:880
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr ""

#: ../duplicity/dup_collections.py:896
#, python-format
msgid "Found backup chain %s"
msgstr ""

#: ../duplicity/dup_collections.py:901
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr ""

#: ../duplicity/dup_collections.py:905
#, python-format
msgid "Found orphaned set %s"
msgstr ""

#: ../duplicity/dup_collections.py:1053
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr ""

#: ../duplicity/dup_collections.py:1275
#, python-format
msgid "File: %s"
msgstr ""

#: ../duplicity/dup_collections.py:1276
#, python-format
msgid "Total number of backup: %d"
msgstr ""

#: ../duplicity/dup_collections.py:1277 ../duplicity/dup_collections.py:1303
msgid "Type of file change:"
msgstr ""

#: ../duplicity/dup_collections.py:1301
#, python-format
msgid " Backup set time: %s"
msgstr ""

#: ../duplicity/dup_collections.py:1302
#, python-format
msgid "Total number of changes: %d"
msgstr ""

#: ../duplicity/dup_collections.py:1303
msgid "File:"
msgstr ""

#: ../duplicity/patchdir.py:71 ../duplicity/patchdir.py:76
#, python-format
msgid "Patching %s"
msgstr ""

#: ../duplicity/patchdir.py:541
#, python-format
msgid "Error '%s' patching %s"
msgstr ""

#: ../duplicity/patchdir.py:617
#, python-format
msgid "Writing %s of type %s"
msgstr ""

#: ../duplicity/selection.py:114
#, python-format
msgid "Skipping socket %s"
msgstr ""

#: ../duplicity/selection.py:118
#, python-format
msgid "Error initializing file %s"
msgstr ""

#: ../duplicity/selection.py:122 ../duplicity/selection.py:160
#: ../duplicity/selection.py:572
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr ""

#: ../duplicity/selection.py:175
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr ""

#: ../duplicity/selection.py:178 ../duplicity/selection.py:199
#: ../duplicity/selection.py:202
#, python-format
msgid "Selecting %s"
msgstr ""

#: ../duplicity/selection.py:255
#, python-format
msgid "                The last file selection option is the filter option %s, which will have no\n"
"                effect as there are no subsequent file selection options. Exiting because\n"
"                this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:265
#, python-format
msgid "                Only these filter mode options were specified:\n"
"                    %s\n"
"                Case sensitive globbing is the default behaviour and so this has no effect.\n"
"                Exiting because this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:328
#, python-format
msgid "                Fatal Error: The file specification\n"
"                    %s\n"
"                cannot match any files in the base directory\n"
"                    %s\n"
"                Useful file specifications begin with the base directory or some\n"
"                pattern (such as '**') which matches the base directory."
msgstr ""

#: ../duplicity/selection.py:337
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr ""

#: ../duplicity/selection.py:354
#, python-format
msgid "Reading files to backup from list %s"
msgstr ""

#: ../duplicity/selection.py:377
#, python-format
msgid "                Files-from list contains the absolute path:\n"
"                    %s\n"
"                All paths specified in a files-from list must be given relative to the backup\n"
"                source path."
msgstr ""

#: ../duplicity/selection.py:387
msgid "                Files-from list specified which contains no files, the backup will be empty as\n"
"                a result. Exiting as this probably isn't what you meant,"
msgstr ""

#: ../duplicity/selection.py:399
#, python-format
msgid "                Last selection expression:\n"
"                    %s\n"
"                only specifies that files be included.  Because the default is to\n"
"                include all files, the expression is redundant.  Exiting because this\n"
"                probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:458
#, python-format
msgid "Reading %s filelist %s"
msgstr ""

#: ../duplicity/selection.py:499
#, python-format
msgid "Error compiling regular expression %s"
msgstr ""

#: ../duplicity/lazy.py:351
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr ""

#: ../duplicity/lazy.py:426
#, python-format
msgid "Error '%s' processing %s"
msgstr ""

#: ../duplicity/lazy.py:436
#, python-format
msgid "Skipping %s because of previous error"
msgstr ""

#: ../duplicity/path.py:106
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr ""

#: ../duplicity/path.py:236 ../duplicity/path.py:297
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr ""

#: ../duplicity/path.py:371
#, python-format
msgid "New file %s"
msgstr ""

#: ../duplicity/path.py:374
#, python-format
msgid "File %s is missing"
msgstr ""

#: ../duplicity/path.py:377
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr ""

#: ../duplicity/path.py:383 ../duplicity/path.py:409
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr ""

#: ../duplicity/path.py:388
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr ""

#: ../duplicity/path.py:396
#, python-format
msgid "Data for file %s is different"
msgstr ""

#: ../duplicity/path.py:404
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr ""

#: ../duplicity/path.py:413
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr ""

#: ../duplicity/path.py:604
#, python-format
msgid "Making directory %s"
msgstr ""

#: ../duplicity/path.py:614
#, python-format
msgid "Deleting %s"
msgstr ""

#: ../duplicity/path.py:623
#, python-format
msgid "Touching %s"
msgstr ""

#: ../duplicity/path.py:630
#, python-format
msgid "Deleting tree %s"
msgstr ""

#: ../duplicity/cli_util.py:49
msgid "Enter 'duplicity --help' for help screen."
msgstr ""

#: ../duplicity/cli_util.py:128
msgid "Running in 'ignore errors' mode due to --ignore-errors.\n"
"Please reconsider if this was not intended"
msgstr ""

#: ../duplicity/cli_util.py:138
msgid "Use of the --asynchronous-upload option is experimental and not safe for production! There are reported cases of undetected data loss during upload. Be aware and periodically verify your backups to be safe."
msgstr ""

#: ../duplicity/cli_util.py:149
msgid "'{val}' is not an int: {str(e)}"
msgstr ""

#: ../duplicity/cli_util.py:163
#, python-brace-format
msgid "'{val} is not a single character."
msgstr ""

#: ../duplicity/cli_util.py:190
#, python-brace-format
msgid "Source should be pathname, not url.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:192
#, python-brace-format
msgid "Argument source_path '{val}' does not exist."
msgstr ""

#: ../duplicity/cli_util.py:198
#, python-brace-format
msgid "Source should be url, not directory.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:204
#, python-brace-format
msgid "Target should be directory, not url.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:209
msgid "Unable to create target dir '{val}': {str(e)}"
msgstr ""

#: ../duplicity/cli_util.py:215
#, python-brace-format
msgid "Target should be url, not directory.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:259
msgid "Verbosity must be one of: digit [0-9], character [ewnid],\n"
"or word ['error', 'warning', 'notice', 'info', 'debug'].\n"
"The default is 3 (Notice).  It is strongly recommended\n"
"that verbosity level is set at 2 (Warning) or higher."
msgstr ""

#: ../duplicity/cli_util.py:394
#, python-brace-format
msgid "Specified archive directory '{archive_dir_path.uc_name}' is not a directory"
msgstr ""

#: ../duplicity/cli_util.py:401
msgid "Encrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/cli_util.py:417
msgid "Hidden dncrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/cli_util.py:427
msgid "Sign key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/argparse311.py:299
msgid "usage: "
msgstr ""

#: ../duplicity/argparse311.py:776
#, python-format
msgid "argument %(argument_name)s: %(message)s"
msgstr ""

#: ../duplicity/argparse311.py:882
msgid ".__call__() not defined"
msgstr ""

#: ../duplicity/argparse311.py:1192
#, python-format
msgid "conflicting subparser: %s"
msgstr ""

#: ../duplicity/argparse311.py:1196
#, python-format
msgid "conflicting subparser alias: %s"
msgstr ""

#: ../duplicity/argparse311.py:1231
#, python-format
msgid "unknown parser %(parser_name)r (choices: %(choices)s)"
msgstr ""

#: ../duplicity/argparse311.py:1291
#, python-format
msgid "argument \"-\" with mode %r"
msgstr ""

#: ../duplicity/argparse311.py:1300
#, python-format
msgid "can't open '%(filename)s': %(error)s"
msgstr ""

#: ../duplicity/argparse311.py:1509
#, python-format
msgid "cannot merge actions - two groups are named %r"
msgstr ""

#: ../duplicity/argparse311.py:1547
msgid "'required' is an invalid argument for positionals"
msgstr ""

#: ../duplicity/argparse311.py:1569
#, python-format
msgid "invalid option string %(option)r: must start with a character %(prefix_chars)r"
msgstr ""

#: ../duplicity/argparse311.py:1587
#, python-format
msgid "dest= is required for options like %r"
msgstr ""

#: ../duplicity/argparse311.py:1604
#, python-format
msgid "invalid conflict_resolution value: %r"
msgstr ""

#: ../duplicity/argparse311.py:1622
#, python-format
msgid "conflicting option string: %s"
msgid_plural "conflicting option strings: %s"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/argparse311.py:1696
msgid "mutually exclusive arguments must be optional"
msgstr ""

#: ../duplicity/argparse311.py:1772
msgid "positional arguments"
msgstr ""

#: ../duplicity/argparse311.py:1788
msgid "show this help message and exit"
msgstr ""

#: ../duplicity/argparse311.py:1819
msgid "cannot have multiple subparser arguments"
msgstr ""

#: ../duplicity/argparse311.py:1871 ../duplicity/argparse311.py:2384
#, python-format
msgid "unrecognized arguments: %s"
msgstr ""

#: ../duplicity/argparse311.py:1971
#, python-format
msgid "not allowed with argument %s"
msgstr ""

#: ../duplicity/argparse311.py:2021 ../duplicity/argparse311.py:2035
#, python-format
msgid "ignored explicit argument %r"
msgstr ""

#: ../duplicity/argparse311.py:2142
#, python-format
msgid "the following arguments are required: %s"
msgstr ""

#: ../duplicity/argparse311.py:2157
#, python-format
msgid "one of the arguments %s is required"
msgstr ""

#: ../duplicity/argparse311.py:2199
msgid "expected one argument"
msgstr ""

#: ../duplicity/argparse311.py:2200
msgid "expected at most one argument"
msgstr ""

#: ../duplicity/argparse311.py:2201
msgid "expected at least one argument"
msgstr ""

#: ../duplicity/argparse311.py:2205
#, python-format
msgid "expected %s argument"
msgid_plural "expected %s arguments"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/argparse311.py:2263
#, python-format
msgid "ambiguous option: %(option)s could match %(matches)s"
msgstr ""

#: ../duplicity/argparse311.py:2327
#, python-format
msgid "unexpected option string: %s"
msgstr ""

#: ../duplicity/argparse311.py:2524
#, python-format
msgid "%r is not callable"
msgstr ""

#: ../duplicity/argparse311.py:2541
#, python-format
msgid "invalid %(type)s value: %(value)r"
msgstr ""

#: ../duplicity/argparse311.py:2552
#, python-format
msgid "invalid choice: %(value)r (choose from %(choices)s)"
msgstr ""

#: ../duplicity/argparse311.py:2630
#, python-format
msgid "%(prog)s: error: %(message)s\n"
msgstr ""

